homework-submission/mobx-week1/src/Componants/TodoList.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { inject, observer } from "mobx-react";
+
+@inject("TodoStore")
+@observer
+class TodoList extends React.Component {
+  state = {
+    clicked: false
+  };
+
+  onEdit = () => {
+    this.setState({
+      clicked: true
+    });
+  };
+  onCancel = () => {
+    this.setState({
+      clicked: false
+    });
+  };
+  updateTodo = (e, id) => {
+    e.preventDefault();
+    const description = this.newDescription.value;
+    const deadline = this.newDeadline.value;
+
+    this.props.TodoStore.updateTodo(id, description, deadline);
+    this.setState({
+      clicked: false
+    });
+  };
+
+  editAction = todo =>
```

Comment:
This method is confused me a bit, about where it's start and where it finished 🙄